lib/tldr.js

```diff
@@ -138,46 +108,40 @@ function printPages(pages, singleColumn) {
 }
 
 function printBestPage(command, options={}) {
-  /* eslint-disable */ // To allow not checking for err
   // Trying to get the page from cache first
-  cache.getPage(command, (err, content) => {
-  /* eslint-enable */
-    if (!content) {
+  cache.getPage(command)
+    .then((content) => {
+      // If found in first try, render it
+      if (content) {
+        checkStale();
+        renderContent(content, options);
+        return exit(0);
```

Comment:
Do we need the `exit(0)` ? Won't it automatically exit with 0 ?

Reply:
You are referring to optional argument?
https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_exit_code

Reply:
I meant, why do we need to call the exit() function at all ? Won't it exit normally by itself ?

Reply:
In this case I'm actually terminating promise chain prematurely so it is required.

Reply:
Got it.